src/client/ui/inputs/Vector3Input.scss

```diff
@@ -14,3 +15,29 @@
   padding: 0 8px;
   color: $text2;
 }
+
+:local(.uniformButton) {
+  display: flex;
+  align-items: center;
+  cursor: pointer;
+
+  &:before {
+    color: $text2;
+    font-size: 10px;
+    padding: 0 6px;
+    @extend .fas;
+    content: fa-content($fa-var-unlock);
```

Comment:
I think this is better represented with a chain link icon. A lock implies that the values cannot change. https://fontawesome.com/icons/link?style=solid

Reply:
Sure, I have no strong opinion. I've seen the lock in Sketch and the link in Figma. Let's go with the link.